apps/expo-go/scripts/fix_rn_location.sh

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-# fixes generate-codegen-artifacts.js (executed in Expo Go's pod install step) resolving react-native to root node_modules/react-native instead of react-native-lab
-# It'd generate incorrect codegen artifacts for react-native
```

Comment:
the error looks like from codegen. does the metro setup help for codegen resolutions?